h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -59,8 +59,8 @@ class SearchBarController extends Controller {
         {
           matchOn: 'url',
           title: 'url:',
-          explanation: `see all annotations on a document URL. * matches any number 
-            of characters and ? matches any single character`,
+          explanation: `see all annotations on a document URL. for domain level search 
```

Comment:
I suggest 'for' should use a capital "F" as it starts a new sentence (the first part of the explanation is a continuation of the one started by the facet name). I'd also suggest that the example URL should be prefixed by "eg." for "For example, ".